services/management/backup/backups_service.go

```diff
@@ -332,6 +332,55 @@ func (s *BackupsService) RemoveScheduledBackup(ctx context.Context, req *backupv
 	return &backupv1beta1.RemoveScheduledBackupResponse{}, nil
 }
 
+// GetLogs returns logs for artifact.
+func (s *BackupsService) GetLogs(ctx context.Context, req *backupv1beta1.GetLogsRequest) (*backupv1beta1.GetLogsResponse, error) {
+	jobs, err := models.FindJobs(s.db.Querier, models.JobsFilter{
+		ArtifactID: req.ArtifactId,
+		Types: []models.JobType{
+			models.MySQLBackupJob,
+			models.MongoDBBackupJob,
+		},
+	})
+	if err != nil {
+		return nil, err
+	}
+	if len(jobs) == 0 {
+		return nil, status.Error(codes.NotFound, "Job related to artifact was not found.")
+	}
+	if len(jobs) > 1 {
+		s.l.Warnf("artifact %s appear in more than one job", req.ArtifactId)
```

Comment:
is that possible? Probably we could error here.

Reply:
Yep, this is possible. For PITR we have one artifact, but multiple jobs.